Mathlib/Analysis/Complex/AbsMax.lean

```diff
@@ -312,6 +312,38 @@ theorem eqOn_closedBall_of_isMaxOn_norm {f : E → F} {z : E} {r : ℝ}
     EqOn f (const E (f z)) (closedBall z r) := fun _x hx =>
   eq_of_isMaxOn_of_ball_subset hd hz <| ball_subset_ball hx
 
+/-- If `f` is differentiable on the open unit ball `{z : ℂ | ‖z‖ < 1}`, and `‖f‖` attains a maximum
```

Comment:
Is `{z : ℂ | ‖z‖ < 1}` really the best way to phrase this, rather than with `ball`?